### PR TITLE
Update GitHub workflows

### PR DESCRIPTION
**Pull request by gnadt**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,20 +19,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version:
-          - '1'
-        os:  [ubuntu-latest, windows-latest, macos-latest]
-        arch:
-          - x64
+        version: [1.6, 1]
```
Comment (Datseris) [inline .github/workflows/ci.yml:22]: I would prefer if this is left as only version `1`. There is no reason to test two versions, latest stable Julia is more than enough. (I am very climate consicous and I believe we all should be avoiding computations that are not necessary)

**Comment by Datseris** [comment]
I've invited you to JuliaMusic and added you admin rights for MusicProcessing.jl by the way.

**Comment by gnadt** [comment]
Hmm I don't know about that, I think it's pretty common to test on the LTS Julia version, this CI is tiny and rarely run anyway

**Comment by Datseris** [comment]
What is the particular functionality that you think really makes a difference to test on LTS? Which particular functions depend on the exact Julia version greater than 1.0? What sort of knowledge would you obtain by testing on LTS that you wouldn't obtain already by testing on latest stable?

**Comment by gnadt** [comment]
Unknown unknowns

**Pull request merged**